content/2020-08-24-newsletter-25.md

```diff
@@ -1,6 +1,6 @@
 +++
-title = "The Embedded Working Group Newsletter - 24"
-date = 2020-02-29
+title = "The Embedded Working Group Newsletter - 25"
+date = 2020-08-24
```

Comment:
I've moved this to be the basis for next week's newsletter, so we don't lose out on the suggestions that have already been added.